### PR TITLE
Playground add permissions to load bundle from bundler

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: android/playground/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,7 +13,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme" 
+        android:usesCleartextTraffic="true" >
```
Comment (RobinCsl) [inline android/playground/src/main/AndroidManifest.xml]: Since that's just the playground, this would never be the case in production, right?

Comment (tbergq) [reply android/playground/src/main/AndroidManifest.xml]: Right, it is only `rnkiwimobile` which is exposed to native application.

**Comment by RobinCsl** [comment]
I can load the `Accout Setting` portion, but the `Hotels` and `Hotels Fragment` crash.

`Hotels` just loads the view, loads the hotel list, and once it's loaded, it crashes.

`Hotels Fragment` returns an error screen:
Screenshot_1575304411

I don't think it's totally related to this PR though. 🤔

**Comment by tbergq** [comment]
I am getting 401's from our maven today 😞

**Comment by tbergq** [comment]
Well, I don't have any of the issues you are describing, and I need this one to finish accounts, so I am going to merge it. 

If the problem you describe arises, we can either fix it, or revert this commit. Anyway, this is not affecting any production code.

**Pull request merged**